docs/getting-started/features/tamper-detection.en.md

```diff
@@ -9,76 +9,62 @@ To secure your Krux device, always verify firmware authenticity before installat
 
-Before being stored in the device’s flash, the *TC Code* is hashed together with the K210 chip’s unique ID and stretched using PBKDF2. This ensures the *TC Code* is not retrievable via a flash dump and can only be brute-forced outside the device if the attacker also has access to the device’s unique ID. By allowing letters, special characters, and running 100k iterations of PBKDF2, brute-forcing the *TC Code* from dumped data becomes more time-consuming and resource-intensive.
+Before being stored in the device’s flash, the *TC Code* is hashed together with the K210 chip’s unique ID and stretched using PBKDF2. This ensures the *TC Code* is not retrievable via a flash dump and can only be brute-forced outside the device if the attacker also has access to the device’s unique ID (UID). By allowing letters, special characters, and running 100k iterations of PBKDF2, brute-forcing the *TC Code* from dumped data becomes more time-consuming and resource-intensive.
 
 ### Enhancing Tamper Detection
 After setting the *TC Code*, you are prompted to fill empty flash memory blocks with random entropy from the camera. This process ensures that attackers cannot exploit unused memory space.
 
-## Tamper Check (TC) Flash Hash - A Tamper Detection Tool
-### Introduction
-
-The "TC Flash Hash" tool enables you to verify if the flash memory content has been altered.
-
-### How It Works
+## Tamper Check Flash Hash (TC Flash Hash) - A Tamper Detection Tool
 
-The tool generates a unique image and four tamper detection words based on a hash of your *TC Code*, the device's UID, and the flash content. The flash memory is divided into two regions:
+The *TC Flash Hash* tool enables you to verify if the device's internal flash memory content has been altered. This tool generates a unique image and two sets of two tamper detection words based on a hash of your *TC Code*, the device's UID, and its internal flash content. The flash memory is divided into two regions:
 
-- **Firmware Region:** Generates the image and the first two words.
+- **Firmware Region:** The area only filled with firmware code. It generates the memorable image and the first set of two words.
 
-- **User's Region:** Generates the last two words.
+- **User's Region:** The area used to stored encrypted mnemonics, settings and TC Code. It generates the last set of two words.
 
 <div style="text-align: center;">
     <img src="../../../img/flash_hash.bmp" alt="TC Flash Hash" width="200"/>
 </div>
 
-*Example: The blue symbol and words 'tail monkey' represent the firmware region, while 'wrestle over' reflects the user region.*
+*Example: The blue symbol and words 'tail monkey' represents the firmware region, while 'wrestle over' user's region.*
```

Comment:
"represent" refers to "words", or "words plus image", so "they represent", no trailing "s"